Error after incident_escalation_path creation

I am creating escalation path. it is mostly copy pasted from the web ui. 
in this path i have a node that must wait for working hours. 
```
[
  {
    id   = "not-working-hours"
    type = "notify_channel"

    notify_channel = {
      targets = [
        {
          type    = "slack_channel"
          id      = "XXX"
          urgency = "high"
        },
      ]

      time_to_ack_weekday_interval_config_id = "default"
      time_to_ack_interval_condition         = "active"
    }
  },
]
```
resource is created but after creation terraform fails with 
```
module.incident_io_escalation_path.incident_escalation_path.this: Creating...
╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵
Stack trace from the terraform-provider-incident_v4.3.2 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xbd10fd]

goroutine 81 [running]:
github.com/incident-io/terraform-provider-incident/internal/provider.(*IncidentEscalationPathResource).toPathModel(0xc0002b8270, {0xc001092400?, 0x1?, 0xc000f0e270?})
        github.com/incident-io/terraform-provider-incident/internal/provider/incident_escalation_path_resource.go:511 +0x5bd
github.com/incident-io/terraform-provider-incident/internal/provider.(*IncidentEscalationPathResource).toPathModel(0xc0002b8270, {0xc001092300?, 0x1?, 0x0?})
        github.com/incident-io/terraform-provider-incident/internal/provider/incident_escalation_path_resource.go:458 +0x1f0
github.com/incident-io/terraform-provider-incident/internal/provider.(*IncidentEscalationPathResource).toPathModel(0xc0002b8270, {0xc001092100?, 0x1?, 0x1747de0?})
        github.com/incident-io/terraform-provider-incident/internal/provider/incident_escalation_path_resource.go:458 +0x1f0
github.com/incident-io/terraform-provider-incident/internal/provider.(*IncidentEscalationPathResource).buildModel(0xc0002b8270?, {{0xc00109c060, 0x1a}, {0xc000f0e240, 0x15}, {0xc0010
92100, 0x1, 0x1}, 0xc000760690})
        github.com/incident-io/terraform-provider-incident/internal/provider/incident_escalation_path_resource.go:434 +0xf9
github.com/incident-io/terraform-provider-incident/internal/provider.(*IncidentEscalationPathResource).Create(0xc0002b8270, {0xf26bb0, 0xc00134cc60}, {{{{0xf2ba18, 0xc000d42330}, {0x
cb3ee0, 0xc000f76ff0}}, {0xf2d6c8, 0xc000e5f360}}, {{{0xf2ba18, ...}, ...}, ...}, ...}, ...)
        github.com/incident-io/terraform-provider-incident/internal/provider/incident_escalation_path_resource.go:338 +0x525
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc0000f8dc0, {0xf26bb0, 0xc00134cc60}, 0xc001319400, 0xc0013193d8)
        github.com/hashicorp/terraform-plugin-framework@v1.2.0/internal/fwserver/server_createresource.go:97 +0x5ab
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0xc0000f8dc0, {0xf26bb0, 0xc00134cc60}, 0xc00134e410, 0xc0013194f0)
        github.com/hashicorp/terraform-plugin-framework@v1.2.0/internal/fwserver/server_applyresourcechange.go:54 +0x4c7
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ApplyResourceChange(0xc0000f8dc0, {0xf26bb0?, 0xc00134cb70?}, 0xc00134e370)
        github.com/hashicorp/terraform-plugin-framework@v1.2.0/internal/proto6server/server_applyresourcechange.go:52 +0x38e
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0xc000650a00, {0xf26bb0?, 0xc00134c1b0?}, 0xc0001e6150)
        github.com/hashicorp/terraform-plugin-go@v0.15.0/tfprotov6/tf6server/server.go:816 +0x56f
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0xda9280, 0xc000650a00}, {0xf26bb0, 0xc00134c1b0}, 0xc0001e6000, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.15.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:419 +0x1a6
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0009ea000, {0xf2c060, 0xc0000e9d40}, 0xc001350000, 0xc0009ec1b0, 0x16d8820, 0x0)
        google.golang.org/grpc@v1.54.0/server.go:1345 +0xdb8
google.golang.org/grpc.(*Server).handleStream(0xc0009ea000, {0xf2c060, 0xc0000e9d40}, 0xc001350000, 0x0)
        google.golang.org/grpc@v1.54.0/server.go:1722 +0x9da
google.golang.org/grpc.(*Server).serveStreams.func1.2()
        google.golang.org/grpc@v1.54.0/server.go:966 +0x8d
created by google.golang.org/grpc.(*Server).serveStreams.func1 in goroutine 51
        google.golang.org/grpc@v1.54.0/server.go:964 +0x226

Error: The terraform-provider-incident_v4.3.2 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,8 @@
 ## Unreleased
 
+- Fixed a panic when using an `incident_escalation_path` with a `notify_channel`
+  node that has a `time_to_ack_interval_condition` set.
+
 ## v4.3.2
 
 - The order of attributes set within an alert source's `template` block is now

--- a/internal/provider/incident_escalation_path_resource.go
+++ b/internal/provider/incident_escalation_path_resource.go
@@ -462,11 +462,16 @@ func (r *IncidentEscalationPathResource) toPathModel(nodes []client.EscalationPa
 			elem.Level = &IncidentEscalationPathNodeLevel{
 				Targets: lo.Map(node.Level.Targets,
 					func(target client.EscalationPathTargetV2, _ int) IncidentEscalationPathTarget {
+						scheduleMode := types.StringNull()
+						if target.ScheduleMode != nil {
+							scheduleMode = types.StringValue(string(*target.ScheduleMode))
+						}
+
 						return IncidentEscalationPathTarget{
 							ID:           types.StringValue(target.Id),
 							Type:         types.StringValue(string(target.Type)),
 							Urgency:      types.StringValue(string(target.Urgency)),
-							ScheduleMode: types.StringValue(string(*target.ScheduleMode)),
+							ScheduleMode: scheduleMode,
 						}
 					}),
 			}
@@ -484,8 +489,7 @@ func (r *IncidentEscalationPathResource) toPathModel(nodes []client.EscalationPa
 				elem.Level.TimeToAckSeconds = types.Int64Value(*value)
 			}
 			if value := node.Level.TimeToAckIntervalCondition; value != nil {
-				elem.Level.TimeToAckIntervalCondition = types.StringValue(
-					string(*node.Level.TimeToAckIntervalCondition))
+				elem.Level.TimeToAckIntervalCondition = types.StringValue(string(*value))
 			}
 			if value := node.Level.TimeToAckWeekdayIntervalConfigId; value != nil && *value != "" {
 				elem.Level.TimeToAckWeekdayIntervalConfigID = types.StringValue(*value)
@@ -495,11 +499,16 @@ func (r *IncidentEscalationPathResource) toPathModel(nodes []client.EscalationPa
 			elem.NotifyChannel = &IncidentEscalationPathNodeNotifyChannel{
 				Targets: lo.Map(node.NotifyChannel.Targets,
 					func(target client.EscalationPathTargetV2, _ int) IncidentEscalationPathTarget {
+						scheduleMode := types.StringNull()
+						if target.ScheduleMode != nil {
+							scheduleMode = types.StringValue(string(*target.ScheduleMode))
+						}
+
 						return IncidentEscalationPathTarget{
 							ID:           types.StringValue(target.Id),
 							Type:         types.StringValue(string(target.Type)),
 							Urgency:      types.StringValue(string(target.Urgency)),
-							ScheduleMode: types.StringValue(string(*target.ScheduleMode)),
+							ScheduleMode: scheduleMode,
 						}
 					}),
 			}
@@ -507,8 +516,7 @@ func (r *IncidentEscalationPathResource) toPathModel(nodes []client.EscalationPa
 				elem.NotifyChannel.TimeToAckSeconds = types.Int64Value(*value)
 			}
 			if value := node.NotifyChannel.TimeToAckIntervalCondition; value != nil {
-				elem.NotifyChannel.TimeToAckIntervalCondition = types.StringValue(
-					string(*node.Level.TimeToAckIntervalCondition))
+				elem.NotifyChannel.TimeToAckIntervalCondition = types.StringValue(string(*value))
 			}
 			if value := node.NotifyChannel.TimeToAckWeekdayIntervalConfigId; value != nil && *value != "" {
 				elem.NotifyChannel.TimeToAckWeekdayIntervalConfigID = types.StringValue(*value)

